developer_manual/app_development/intro.rst

```diff
@@ -13,6 +13,35 @@ After :doc:`you've set up the development environment <../getting_started/devenv
 
 Then create a skeleton app in the `app store <https://apps.nextcloud.com/developer/apps/generate>`_. This doesn't publish anything on the appstore yet, it just gives you a download.
 
+Edit an existing app
+--------------------
+
+Alternatively, if you would like to contribute to an existing app instead of creating a new one, first :doc:`set up the development environment <../getting_started/devenv>`, then create an `apps-extra` folder in the Nextcloud root directory::
+
+    cd /var/www/nextcloud/apps-extra
+    
+You can then configure Nextcloud to run apps from this directory, by changing your `app_paths` system config in your `config.php`
+
+    'apps_paths' => 
+    array (
+      0 => 
+      array (
+        'path' => '/var/www/html/apps',
+        'url' => '/apps',
+        'writable' => false,
+      ),
+      1 => 
+      array (
+        'path' => '/var/www/html/apps-extra',
+        'url' => '/apps-extra',
+        'writable' => false,
```

Comment:
I'm guessing that was on purpose to avoid Nextcloud overwriting data in those cloned apps git repo